### ISSUE TITLE
Is it possible to translate this mode into Korean?

### ISSUE THREAD

**Issue by So1207** [Happydragun4now/DragoonCapes#16]
I created a translation file because I wanted to use it in Korean, but I don't know how to share it.
I translated it through a translator, and there are some parts that are a bit clumsy.
Still, it's not a bad read. If there is a better translation, feel free to improve it! Do whatever you want..
I just want to use it as a Korean translation. Could you please add it?

[Korean2.json](https://github.com/Happydragun4now/DragoonCapes/files/14826379/Korean2.json)

https://valheim-modding.github.io/Jotunn/data/localization/translations/Korean.html
I modified some notations by referring to this page. 2 in the title is the modified version. I don't think the translation of the attack options was bad, but the item descriptions may be of poor translation quality.

**Comment by Happydragun4now** [comment]
Ok I'm going to trust your translation and hopefully you didn't put anything raunchy in there lmao.

It will be in the next update, thanks bro

**Comment by So1207** [comment]
I didn't know you would reply so quickly!
I started translating because I wanted to introduce your mod to my Korean friends. Thank you for making such a great mod. 

This was my first attempt and I learned a lot of ways to edit it even after sending it to you. This might be a bit annoying...I'm sorry. But I wanted to preserve the original as much as possible. I didn't want to ruin the immersive content.

Although the meaning may have changed in the process of refining English to Korean, at least one thing can be said clearly. I tried my best to capture it as is! My skills are too poor to include any sensational content. Do not worry!

[Korean3.json](https://github.com/Happydragun4now/DragoonCapes/files/14831558/Korean3.json)

When compared, there are some differences. Much better than before!
